app/src/androidTestKiwix/java/org/kiwix/kiwixmobile/tests/activities/BookmarksActivityTest.java

```diff
@@ -0,0 +1,32 @@
+package org.kiwix.kiwixmobile.tests.activities;
```

Comment:
Please add javadoc package documentation to these various new files. Mention why this test exists (ditto for the other new tests). I've also added some comments later on in this review (on the magic 250 value for sleeps - it's helpful to mention how you reached this number rather than others e.g. 100 was too short and ... failed at times, 10000 was too long because .... I ended up picking 250 as it seemed to wait long enough for the UI to settle without delaying the tests, blah blah)